src/thing_stats.c

```diff
@@ -321,20 +321,18 @@ long compute_creature_kind_score(ThingModel crkind, unsigned short crlevel)
 /* Computes max health of a creature on given level. */
 HitPoints compute_creature_max_health(HitPoints base_health, unsigned short crlevel)
```

Comment:
imp.cfg has documented the range of base health. You would need to remove or correct that when you change this code.

Reply:
yes